# Try something!

fun main():no_value
{
    try:
    {
        writeln(8)
    }
    catch():
    {
        throw("What happened?")
    }
}
